### DISCUSSION TITLE
naive-ui与unocss的冲突

### DISCUSSION THREAD

**Discussion by 379949990** [antfu/vitesse#357] [General]
我在初始化vitesse项目后尝试引入naive-ui，引入后后我首先创建了一个基础的n-button组件，发现button的background-color被重置为transparent使得naive-ui的button样式没有生效，如图：
<img width="652" alt="image" src="https://user-images.githubusercontent.com/66375788/168445084-ec30b739-7139-4b7d-aa4c-a5078ff1470a.png">
<img width="501" alt="image" src="https://user-images.githubusercontent.com/66375788/168444827-b6681bac-2f45-4c82-8e55-8472dce10adf.png">
我使用naive-ui文档中提供的方法解决了样式覆盖这一问题：
<img width="726" alt="image" src="https://user-images.githubusercontent.com/66375788/168444949-860181e3-84df-485d-a67f-cf5c54b43bb9.png">
随后我发现unocss(默认预设)提供的的"m-3"失效了，注释掉上图的三行代码后"m-3"又恢复正常了，请问我该怎么做？

**Comment by BreakingLead** [comment]
删了unocss

**Comment by ghost** [comment]
_// In App.vue_

```
<script setup lang="ts">
import { darkTheme, lightTheme } from 'naive-ui'
</script>

<template>
  <n-config-provider :theme="isDark ? darkTheme : lightTheme">
    <RouterView />
  </n-config-provider>
</template>
```
